resourcemgr/resourcemgr.c

```diff
@@ -2939,6 +2939,7 @@ TSS2_RC InitResourceMgr( int debugLevel)
     rval = GetCommands( resMgrSysContext, &supportedCommands );
     if( rval != TPM_RC_SUCCESS )
     {
+        rval += TSS2_RESMGR_ERROR_LEVEL;
```

Comment:
We should not use += to change the error level, it will cause problem if there already be a non-zero error level. I suggest using: 
rval = (rval & (~TSS2_ERROR_LEVEL_MASK)) + TSS2_RESMGR_ERROR_LEVEL;;